src/components/stepforms/widgets/List.vue

```diff
@@ -11,6 +11,8 @@
             :is="widget"
             v-bind="componentProps"
             :value="child.value"
+            :available-variables="availableVariables"
+            :variable-delimiters="variableDelimiters"
```

Comment:
Why not store `availableVariables` et `variableDelimiters` into the vuex store ? In my understanding, all widgets will need these variables and they are global.

Reply:
@rhuille that's already the case but we want to pass it through all components ( store variables only keeped in steps) to allow to use components with props rather than with store